aries-site/src/data/structures/components.js

```diff
@@ -1486,6 +1486,7 @@ export const components = [
   },
   {
     name: 'DataTable',
+    accessibility: 'Failed WCAG 2.2 AA',
```

Comment:
Seems like we are failing A level
```suggestion
    accessibility: 'Failed WCAG 2.2 A',
```